Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Mesh/MeshFeatureProcessor.h

```diff
@@ -132,11 +134,6 @@ namespace AZ
             // which are turned into instance draw calls after culling
             AZStd::vector<PostCullingInstanceDataList> m_postCullingInstanceDataByLod;
             
```

Comment:
Do we not need m_updateDrawPacketEventHandlersByLod anymore?

Reply:
Correct. we use the m_associateIntances to replace the event/handlers